features/frontend/checkout_finalize.feature

```diff
@@ -42,8 +42,7 @@ Feature: Checkout finalization
           And I select the "Dummy" radio button
           And I press "Continue"
          When I click "Place order"
-         Then I should be on the store homepage
-          And I should see "Thank you for your order!"
+         Then I should see "Thank you sylius@example.com"
```

Comment:
We should use a name here. :) If I recall correctly, we generate it randomly - we can hardcode a specific name for now. These scenarios will be changed, but we should definitely use customer name on the thank page, not e-mail.